UI/src/view/point_asset/trafficSignForm.js

```diff
@@ -488,7 +544,7 @@
       return sorted;
     };
 
-    var renderAdditionalPanels = function (property, collection) {
+    var renderAdditionalPanels = function (property, collection, asset) {
```

Comment:
So I have been seeing this "asset" variable/parameter everywhere in this file, can this be put in a global variable of this form?

Reply:
Done